src/content/app/genome-browser/components/browser-cog/BrowserCogList.tsx

```diff
@@ -37,6 +37,8 @@ import BrowserCog from './BrowserCog';
 
 import styles from './BrowserCogList.scss';
 
+export type CogList = Record<string, number>;
```

Comment:
I don't believe this type is used anywhere?

Reply:
Ah another one of those I couldn't revisit. Thanks for spotting this.

Reply:
✅